Source/ACE.Server/Factories/LootGenerationFactory.cs

```diff
@@ -1053,95 +1054,37 @@ private static int GetSpellDistribution(TreasureDeath profile, out int numMinors
-                    if (nonCantripChance <= 50)
-                        numNonCantrips = 4;
-                    else
-                        numNonCantrips = 5;
-
-                    numMajors = GetNumMajorCantrips(profile);
-                    break;
-
-                case 4:
-                    // 5-6, major and minor
-                    if (nonCantripChance <= 50)
-                        numNonCantrips = 5;
-                    else
-                        numNonCantrips = 6;
-
-                    numMajors = GetNumMajorCantrips(profile);
-                    break;
-
-                case 5:
-                    // 5-7 major and minor
-                    if (nonCantripChance <= 50)
-                        numNonCantrips = 5;
-                    else if (nonCantripChance <= 90)
-                        numNonCantrips = 6;
-                    else
-                        numNonCantrips = 7;
-
-                    numMajors = GetNumMajorCantrips(profile);
-                    break;
-
-                case 6:
-                    // 6-7, minor(4 total) major(2 total)
-                    if (nonCantripChance <= 50)
-                        numNonCantrips = 6;
-                    else
-                        numNonCantrips = 7;
-
-                    numMajors = GetNumMajorCantrips(profile);
-                    break;
-
-                case 7:
-                    /// 6-7, minor(4), major(3), epic(4)
-                    if (nonCantripChance <= 50)
-                        numNonCantrips = 6;
-                    else
-                        numNonCantrips = 7;
-
-                    numMajors = GetNumMajorCantrips(profile);
-                    numEpics = GetNumEpicCantrips(profile);
-                    break;
-
-                default:
-                    // 6-7, minor(4), major(3), epic(4), legendary(2)
-                    if (nonCantripChance <= 50)
-                        numNonCantrips = 6;
-                    else
-                        numNonCantrips = 7;
-
-                    numMajors = GetNumMajorCantrips(profile);
-                    numEpics = GetNumEpicCantrips(profile);
-                    numLegendaries = GetNumLegendaryCantrips(profile);
-                    break;
-            }
+            numMajors = GetNumMajorCantrips(profile);
+            numEpics = GetNumEpicCantrips(profile);
+            numLegendaries = GetNumLegendaryCantrips(profile);
+
+            //  Fixing the absurd amount of spells on items - HQ 6/21/2020
+            //  From Mags Data all tiers have about the same chance for a given number of spells on items.  This is the ratio for magical items.
+            //  1 Spell(s) - 46.410 %
+            //  2 Spell(s) - 27.040 %
+            //  3 Spell(s) - 17.850 %
+            //  4 Spell(s) - 6.875 %
+            //  5 Spell(s) - 1.525 %
+            //  6 Spell(s) - 0.235 %
+            //  7 Spell(s) - 0.065 %
+
+            if (nonCantripChance <= 46410)
```

Comment:
is this code no longer based on tier?

Reply:
By memory, agree that end of retail 7+ spell items were rare.  But I recall ~4 or 5 spell items being very common.  What was the data used to decide upon the distribution where a 5-spell item should be a 1.525% find?

Reply:
From retail data that was collected.  Thats where the data came from

Reply:
Got it.  Did that include any banding by Tier?  For example, was it the same distribution in PCAP for Drudgehideout item with Lvl 1 spells and Dark Isle item with Lvl 8's?